### PR TITLE
bunch of minor issues

### PR DESCRIPTION
**Pull request by jordansread**
image

tooltips, capitalization, etc

### CODE REVIEW

**Comment by jordansread** [comment]
no units or context for the mouseover, but this gets us to diagnostic

File: layout/js/map.js [hunk 1]
```diff
@@ -50,10 +47,28 @@ var animate_bars = function(data) {
...
+      } else {
+        bar.css('opacity','1.0');
+      }
     }
   });
 };
 
+
+var setStateValue = function(state) {
+  $.get( "js/scaleFactors.json", function( data ) {
```
Comment (jiwalker-usgs) [inline layout/js/map.js]: We don't want an async call that gets called this often (it is probably hitting the cache which is why it works alright).  We could add in a promise from the onReady so we know we have the data (or just check that `transformData` is not undefined).

File: layout/js/map.js [hunk 2]
```diff
@@ -50,10 +47,28 @@ var animate_bars = function(data) {
...
+      var allData = transformData["totState"][year][category];
+      for (var i = 0; i < allData.length; i++) {
+        if (allData[i]['state_name'] === state){
+          stateVal = allData[i]['value'];
+          break;
```
Comment (jiwalker-usgs) [inline layout/js/map.js]: To avoid a `break` here, I'd probably wrap this bit in a function and return stateVal.

Comment (jiwalker-usgs) [reply layout/js/map.js]: One other option would be to build a more efficient lookup when initially getting the data.  It is probably not great to search through the entire state list each time.

Comment (jordansread) [reply layout/js/map.js]: Yeah, I wasn't sure how to index the json like this to get what I wanted

File: layout/js/map.js (hunk 1 above)
Comment (jordansread) [reply layout/js/map.js]: Geez....brutal, I didn't know that was round-tripping for a file. Ouch

Comment (jordansread) [reply layout/js/map.js]: BTW, I haven't merged this, so it is just local right now (so I didn't crash anything). But I could use some help refactoring this so it makes sense.

Comment (jiwalker-usgs) [reply layout/js/map.js]: Yeah, $.get goes out and grabs a file.

File: layout/js/map.js (hunk 2 above)
Comment (jordansread) [reply layout/js/map.js]: Plus this gets run each time the mouse is moving over a state - not just when the state changes

File: layout/js/map.js [hunk 3]
```diff
@@ -17,7 +18,7 @@ var colors = {
 var animate_resize_map = function(data) {
   var color = colors[category];
   $.each(data, function(index, val) {
-    var scale = val.scaleFactor;
+    var scale = Math.sqrt(val.scaleFactor);
```
Comment (jordansread) [inline layout/js/map.js:26]: we think this fixes #95 and likely part of #82

**Pull request merged**